frontend/src/pages/question/QuestionNew.jsx

```diff
@@ -114,30 +88,34 @@ export default function QuestionNew() {
       return;
     }
 
-    // Redirect to created question
-    history.push(`/question/${questionId}`);
-    displayAlert('info', "Fetching answer, we will let you know when it's ready.");
+    pageStatus.setLoading('Fetching answer, this may take a while');
 
     // Start the process of getting an answer and display to user when done
-    getAnswerAndDisplayAlert(questionId);
+    await fetchAnswer(questionId);
+
+    // Redirect to created question
+    history.push(`/question/${questionId}`);
```

Comment:
The answer id is returned from API.queryDispatcher.getAnswer. We should move this to the end of fetchAnswer() and include the answer id in the route. Just to make sure we go to the correct answer.

Reply:
If we wanted to be fancy, once we get the answer back, we could check if the user is still on this page (maybe by looking at location). If they're still here, send them to the answer page, if they're somewhere else, just show the success alert.